bundle/Resources/views/ContentType/field_types.html.twig

```diff
@@ -183,18 +183,34 @@
         {{- form_widget(form.isMultiple) -}}
     </div>
 
-    <div class="ezselection-settings options{% if group_class is not empty %} {{ group_class }}{% endif %}">
+    <div id="{{form.options.vars.id}}-ezselection-root-options-list"
+         class="ezselection-settings options{% if group_class is not empty %} {{ group_class }}{% endif %}"
+         data-prototype="{{ ( form_widget(form.options.vars.prototype) ~ ' ' ~
```

Comment:
@dpobel Do you mean the concept of making prototype html and store it in an html attribute, and use javascript to write it into the document? That's the way Symfony recommends it:
http://symfony.com/doc/current/cookbook/form/form_collections.html#cookbook-form-collections-new-prototype

Or do you mean my implementation of it? That may of course be ugly :) I am concatenating more stuff manually. It might be possible to put all of it in a twig template and get Symfony to accept that as prototype instead of the default.